source/user-manual/capabilities/wazuh-logtest/logtest-configuration.rst

```diff
@@ -0,0 +1,39 @@
+.. Copyright (C) 2020 Wazuh, Inc.
+
+.. _logtest_configuration:
+
+Configuration
+=============
+
+Wazuh-Logtest is a functionality provided by the manager, whose work parameters are configured in the :ref:`ossec.conf <reference_ossec_conf>` file in the section :ref:`rule_test <reference_ossec_rule_test>`.
+
+By default, the configuration is:
+
+.. code-block:: xml
+
+	<rule_test>
+	    <enabled>yes</enabled>
+	    <threads>1</threads>
+	    <max_sessions>64</max_sessions>
+	    <session_timeout>15m</session_timeout>
+	</rule_test>
+
+And it has the following configuration parameters
+
++-----------------+----------------------------------------------+----------------+---------------------------------+
+|    Parameter    |                Description                   |    Default     |    Values allowed               |
++=================+==============================================+================+=================================+
+| enabled         | Determine if logtet is enabled  or disabled  |      yes       | yes/no                          |
++-----------------+----------------------------------------------+----------------+---------------------------------+
+| threads         | Number of Wazuh-Logtest threads              |                | a number between 1 and 128,     |
+|                 |                                              |       1        | or ``auto`` to create one       |
+|                 |                                              |                | thread per CPU                  |
++-----------------+----------------------------------------------+----------------+---------------------------------+
+| max_sessions    | Number of users connected simultaneously     |      64        | A number between 1 and 500      |
++-----------------+----------------------------------------------+----------------+---------------------------------+
+| session_timeout | Time interval in which a client must remain  |                | A positive number that should   |
```

Comment:
Change it to` A positive number that should contain a suffix character indicating a time unit, such as, s (seconds), m (minutes), h (hours)`
View string_op.c:wlogtest and LOGTEST_LIMIT_SESSION_TIMEOUT, 
The max value is 365 days